R/utils.R

```diff
@@ -761,3 +761,20 @@ calc_sd <- function(x) {
   }
 }
 
+
+
+#' safe wrapper of Sys.getenv()
+#' 
+#' So far the helper is needed to handle env vars containing `:` 
+#' for which the backslash  is automatically added in some contexts
+#' and R could not get the original value for these env vars.
+#' 
+#' @param x string with the name of the environemntal variable
+#' @param ... additional params for Sys.getenev
+#' @keywords utils
```

Comment:
gDRutils does not have this `@keywords`

Reply:
Good catch. It was fixed in ed15219f4acbf3cb4e57518c6b60d3bfe408464d.

Reply:
✔️